tests/integration-tests/src/test/resources/features/oid4vci/manage_issuer.feature

```diff
@@ -1,19 +1,40 @@
 @oid4vci
 Feature: Manage OID4VCI credential issuer
 
-Scenario: Successfully create credential issuer
+  Scenario: Successfully create credential issuer
     When Issuer creates an oid4vci issuer
     Then Issuer sees the oid4vci issuer exists on the agent
     And Issuer sees the oid4vci issuer on IssuerMetadata endpoint
 
-Scenario: Successfully update credential issuer
+  Scenario: Successfully update credential issuer
     Given Issuer has an existing oid4vci issuer
     When Issuer updates the oid4vci issuer
     Then Issuer sees the oid4vci issuer updated with new values
     And Issuer sees the oid4vci IssuerMetadata endpoint updated with new values
 
-Scenario: Successfully delete credential issuer
+  Scenario: Successfully delete credential issuer
     Given Issuer has an existing oid4vci issuer
     When Issuer deletes the oid4vci issuer
     Then Issuer cannot see the oid4vci issuer on the agent
     And Issuer cannot see the oid4vci IssuerMetadata endpoint
+
+  Scenario Outline: Create issuer with <description> expect <httpStatus> response
+    When Issuer tries to create oid4vci issuer with '<id>', '<url>', '<clientId>' and '<clientSecret>'
+    Then Issuer should see the oid4vci '<httpStatus>' http status response with '<errorDetail>' detail
+    Examples:
+      | id                                   | url                | clientId | clientSecret | httpStatus | errorDetail                      | description        |
+      | null                                 | null               | null     | null         | 400        | authorizationServer.url          | null values        |
+      | null                                 | malformed          | id       | secret       | 400        | Invalid URL                      | malformed url      |
```

Comment:
`malforrmed` is actually a valid URL. We should repurpose this case to `Disallow relative URL`

Reply:
@patlo-iog, I saw you already Disallow the relative URL is the latest PR